### PR TITLE
Landing page requires new structure

### PR DESCRIPTION
**Pull request by giannamoth**
Hei Hei :)

Check it out, especially I need you to look at two things:

1. Does the upload-button as a type=text  work? If not I'm gonna put it back as file
2. Absolutely necessary to change the main page to be the index one, and keep the about as a secondary one. This is because the overall Style system is linked in the index.css and it is generating conflicts by being the cond one. And secondly, the UX team is planning to have the index.ejs as landing page and not about.
Danke :)

### CODE REVIEW

**Comment by vagechirkov** [comment]
@giannamoth Great work! Thank you! I will add some changes today.

File: node_app/client/views/generate/index.ejs [hunk 1]
```diff
@@ -1,70 +1,91 @@
...
-                    </div>
-                    <div>
-                        <label for="classicRange" class="form-label">Classic</label>
-                        <input name="classicRange" type="range" class="form-range" min="0" max="6"
-                               id="classicRange">
+            -->
+                    <div id="uploadFileInput">
+                        <span class="input-group-text btn btn-primary" id="inputGroup-sizing-default">Choose a
+                            file</span>
+                        <input type="text" name="audio" class="form-control" aria-label="Sizing example input"
```
Comment (vagechirkov) [inline node_app/client/views/generate/index.ejs]: @giannamoth input type should be `type="file"`, it won't work otherwise, unfortunately :(

Comment (giannamoth) [reply node_app/client/views/generate/index.ejs]: hehe I was imagine so 😄  No worries, I'm gonna set it back to what it was 😎

**Comment by vagechirkov** [comment]
The design looks fantastic, but I have some minor comments.

First, do you think it is possible to restructure this page somehow so that we have the full generate form on one screen? Maybe this is just my screen though

<details>

image

</details>


Second, if you go to `http://127.0.0.1:3030/generate/results`, do you think it is possible to glue this background gradient change to the bottom of the screen?


<details>

image

</details>

**Comment by vagechirkov** [comment]
And spectrogram disappears when I change the web app size. I am not sure why 

middle/small

<details>

image

</details>

large

<details>

image

</details>

**Comment by giannamoth** [comment]
1-Not sure to understand what do you mean with a "full generate form"
2-So the gradient in the  `http://127.0.0.1:3030/generate/results ` happen because of the wrong allocation of the index page, as not the main page.
3- good one!!! I'm gonna investigate about this weird behaviour of the moving elements

**Comment by giannamoth** [comment]
Screen Shot 2022-01-25 at 15 19 17
I have the about.ejs allocated outside the generate folder. Is this correct?

**Comment by giannamoth** [comment]
And I'm not sure to be able to make the changes before the reshuffling of the index/about the landing page. 
I have a notification that it has happened but it doesn't look like on my machine :/
HOw can I be sure that is right? (sry for the most likely dumb question)

**Comment by vagechirkov** [comment]
Hey, you need to pull the changes to your local machine first. I believe we are working on the `giannaDevelop` branch in this pull request, so you need to pull changes from this branch

**Comment by vagechirkov** [comment]
> 1-Not sure to understand what do you mean with a "full generate form"

I mean that I need to scroll to see the Generate button. This is not a problem, just a short notice

**Comment by giannamoth** [comment]
achsooo, well I can see it on my machine, it probably depends on the device you are using

**Comment by giannamoth** [comment]
I've pulled, but it doesn't look so much different from before .... and what about the  `about.ejs ` location?

**Comment by vagechirkov** [comment]
> 2-So the gradient in the http://127.0.0.1:3030/generate/results happen because of the wrong allocation of the index page, as not the main page.

I am not sure that I understood what do you mean, sorry :(

**Comment by giannamoth** [comment]
I think that the weird lines happened because the index was not the main landing page and so the CSS files had a different priority-impact on the visual aesthetic (just an assumptions)

**Comment by vagechirkov** [comment]
> I have the about.ejs allocated outside the generate folder. Is this correct?

I am a bit imprecise about the route naming, you are right. The idea is that the about page is one of the general pages. I might need to move index.ejs to the root too. I will think about it, but it should not be a problem to refactor this at any time

**Comment by giannamoth** [comment]
Yes, I actually really need that! I think that is gonna make more sense from a page hierarchy point of view

**Comment by vagechirkov** [comment]
> I think that the weird lines happened because the index was not the main landing page and so the CSS files had a different priority-impact on the visual aesthetic (just an assumptions)

Ah, alright, let's keep it! This is not critical for now

**Comment by vagechirkov** [comment]
> Yes, I actually really need that! I think that is gonna make more sense from a page hierarchy point of view

done :)

**Comment by giannamoth** [comment]
Screen Shot 2022-01-25 at 15 48 02
It is death now :(

**Comment by vagechirkov** [comment]
Should work now:)

**Comment by giannamoth** [comment]
nope

**Comment by vagechirkov** [comment]
> nope

Did you pull the changes?

The [deployed version](https://noise-enricher.herokuapp.com/) works :)

**Comment by giannamoth** [comment]
yes I did. I've dann decide to change the machine, (from work pc to personal pc) and now it works...haha sometimes I really dunno why things happen xD
SO yess, all regular now 😎

**Pull request merged**